<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>inputDataToDb</class>
 <widget class="QWidget" name="inputDataToDb">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>384</width>
    <height>326</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>331</width>
     <height>271</height>
    </rect>
   </property>
   <property name="title">
    <string>Данные</string>
   </property>
   <widget class="QLineEdit" name="firstNameLineEdit">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>90</y>
      <width>131</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="addDataPushButton">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>230</y>
      <width>251</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Добавить</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>130</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Дата рождения   --&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>180</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Пол                       --&gt;</string>
    </property>
   </widget>
   <widget class="QComboBox" name="sexComboBox">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>180</y>
      <width>131</width>
      <height>24</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Мужской</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Женский</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>90</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Имя                      --&gt;</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lastNameLineEdit">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>40</y>
      <width>131</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QDateEdit" name="dateOfBirthdayEdit">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>130</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>40</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Фамилия             --&gt;</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
